server/database/migrations/20190602223101-create-lending-history.js

```diff
@@ -24,13 +19,32 @@ module.exports = {
         },
       },
       bookId: {
+        allowNull: false,
         type: Sequelize.INTEGER,
         onDelete: 'CASCADE',
         references: {
           model: 'Books',
           key: 'id',
         },
       },
+      charge: {
+        allowNull: false,
+        type: Sequelize.INTEGER
+      },
+      type: {
+        allowNull: false,
+        type: Sequelize.STRING,
+        defaultValue: 'borrowed'
+      },
+      duration: {
+        allowNull: false,
+        type: Sequelize.STRING,
+        defaultValue: '1 month'
```

Comment:
does this value change?

Reply:
Yes, in case a patron chooses to extend the duration